pkg/gitrepo/repo_test.go

```diff
@@ -12,6 +12,8 @@ import (
 // Test_New_optionalURL tests if proper URL from origin branch is taken from
 // existing repository if none is specified.
 func Test_New_optionalURL(t *testing.T) {
+	t.Parallel()
```

Comment:
To speed things up a little. We pull github repository in every tests. This shaves a second or two.

Reply:
God shave the queen.